src/views/InvolvementProfile/index.jsx

```diff
@@ -411,8 +400,8 @@
             <Grid item align="center">
               {ActivityBlurb && <Typography>{ActivityBlurb}</Typography>}
               {ActivityURL?.length !== 0 && (
-                <Typography>
-                  <a href={ActivityURL} className="gc360_text_link" style={{ fontWeight: 'bold' }}>
+                <Typography className={styles.activityURL}>
+                  <a href={ActivityURL} className="gc360_text_link">
```

Comment:
@EjPlatzer we got this warning about the involvement profile website link being potentially vulnerable to cross-site scripting.  It doesn't look like there is any code in the frontend to prevent scripting, so might need to be looked into.

Reply:
This is the true in the sense that a user with sufficient permissions to change the ActivityURL (i.e. a group admin or a site admin) can edit it to be a malicious URL (see the current URL for the 360 involvement in Train as an example).

There's a question of tradeoffs here - the goal of this field is to display a user-provided URL, which will never truly be safe because a malicious user can input a URL that redirects to e.g. a Man In The Middle phishing page. On the other hand, it's possible we could sanitize some of the most obviously harmful URLs (although sanitization is notoriously difficult). It's worth exploring more, but I would also say this isn't an imminent security threat for our threat model.